internal/proxy/anthropic_hang_fixes_test.go

```
@@ -0,0 +1,296 @@
+			StatusCode: http.StatusOK,
+			Body:       pr,
+			Header:     http.Header{"Content-Type": []string{"application/vnd.amazon.eventstream"}},
+		}, nil
+	})
+	s := fableStreamTestServer(rt)
+	done := make(chan struct{})
+	var resp *http.Response
+	var respErr error
+	go func() {
+		defer close(done)
+		resp, respErr = s.claudeFableBedrockResponse(context.Background(), fableStreamTestBody)
```

golangci-lint (2.12.2)

[error] response body must be closed

(bodyclose)